docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -226,7 +235,9 @@ These files cannot be edited by the standard tooling.
 
 ## NuGet defaults file
 
-The `NuGetDefaults.Config` file exists to specify package sources from which packages are installed and updated, and to control the default target for publishing packages with `nuget push`. Because administrators can conveniently (using Group Policy, for example) deploy consistent `NuGetDefaults.Config` files to developer and build machines, they can ensure that everyone in the organization is using the correct package sources rather than nuget.org.
+The `NuGetDefaults.Config` is uncommon and can only specify package sources from which packages are installed and updated, or control the default target for publishing packages with `nuget push`.
+
+Because administrators can conveniently (using Group Policy, for example) deploy consistent `NuGetDefaults.Config` files to developer and build machines, they can ensure that everyone in the organization is using the correct package sources, whether or not that includes nuget.org.
```

Comment:
"ensure" is a strong word. Can't `<clear/>` still invalidate this config? I wonder if we should drop this paragraph entirely.

Reply:
No, when I was working on some config file changes for audit sources, I discovered that sources defined in `NuGetDefaults.Config` cannot be removed or disabled.  This should be documented.

I also don't like "correct package sources". I'd say "consistent package sources", "company package sources" or something similar.